protobuf/proto/crux.proto

```diff
@@ -83,6 +83,7 @@ service CruxDeployment {
   rpc GetDeploymentSecrets(DeploymentListSecretsRequest) returns (common.ListSecretsResponse);
   rpc CopyDeploymentSafe(IdRequest) returns (CreateEntityResponse);
   rpc CopyDeploymentUnsafe(IdRequest) returns (CreateEntityResponse);
+  rpc PreDeploymentCheck(IdRequest) returns (common.Empty);
```

Comment:
We missed the chance to talk about this last time.

Reply:
Today then 😇